NuGet.Services.Dashboard.Operations.Tools/app.config

```diff
@@ -53,5 +57,9 @@
     <add key="MailRecepientAddress" value="" />
     <add key="UsePagerDuty" value="" />
     <add key="PagerDutyServiceKey" value="" />
+    <add key="PagerDutySev1ServiceKey" value="" />
+    <add key="keyVault-VaultName" value="nugetdevkeyvault" />
+    <add key="keyVault-ClientId" value="0d0b9c1e-8d58-4821-8937-039d5d7fee0b" />
+    <add key="keyVault-CertificateThumbprint" value="CD65005F17721835F9E036A2FCAA4A063F3CDE97" />
```

Comment:
this config is incorrect for dashboard (should use prod KeyVault), and should not be in github. please clear the values